### ISSUE TITLE
mariadb: fails to build on arm_cortex-a7_neon-vfpv4

### ISSUE THREAD

**Issue by PolynomialDivision** [openwrt/packages#17073]
@micmac1

```
Package mariadb-server is missing dependencies for the following libraries:
liburing.so.2
```

https://downloads.openwrt.org/snapshots/faillogs/arm_cortex-a7_neon-vfpv4/packages/mariadb/compile.txt

**Comment by PolynomialDivision** [comment]
Should be fixed by: https://github.com/openwrt/packages/pull/17064

**Comment by micmac1** [comment]
And you're pinging me... because? ;-)

**Comment by PolynomialDivision** [comment]
You are maintainer?

**Comment by micmac1** [comment]
Nope. Check the Makefile.

**Comment by cotequeiroz** [comment]
Closed by #17064
